src/nuklear_style.c

```diff
@@ -193,6 +201,8 @@ nk_style_from_table(struct nk_context *ctx, const struct nk_color *table)
     toggle->border_color    = nk_rgba(0,0,0,0);
     toggle->border          = 0.0f;
     toggle->spacing         = 4;
+    toggle->color_factor    = 1.0f;
+    toggle->disabled_factor = 0.5f;
```

Comment:
Lots of setting of `0.5f`. Perhaps might benefit from adding a `NK_WIDGET_DISABLED_FACTOR` or something so that it's easy to change globally?

Reply:
Very true, I will fix that :)